src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx

```diff
@@ -87,6 +112,94 @@ export class CytoscapeReactWrapper extends React.Component<CytoscapeReactWrapper
       this.cy = null;
     }
   }
+
+  private buildContextMenus() {
+    const makeTippyForNode: any = (node: any) => {
+      return tippy(node.popperRef(), {
+        content: (() => {
+          // Remember we on Tippy.js which is html over the cytoscape canvas -- not React JSX
```

Comment:
I wonder if ReactDOM.render could be used for this.

See
https://reactjs.org/docs/rendering-elements.html#rendering-an-element-into-the-dom

Reply:
Oh good idea, I didn't think about that.

Reply:
I hated having the plain html in there but didn't see any options since tippy is html over canvas, but this idea makes sense.

Reply:
I wasn't able to make the ReactDOM.render() work. Maybe we can try again in another PR.